_unittest/test_00_EDB.py

```diff
@@ -635,3 +635,15 @@ def test_75_primitives_area(self):
     def test_76_short_component(self):
         assert self.edbapp.core_components.short_component_pins("EU1", width=0.2e-3)
         assert self.edbapp.core_components.short_component_pins("U10", ["2", "5"])
+
+    def test_77_convert_padstack_pad_to_rectangle(self):
+        edb = Edb(
+            edbpath=os.path.join(local_path, "example_models", "padstacks.aedb"),
+            edbversion=desktop_version,
+            isreadonly=True,
+        )
+        for i in range(7):
+            padstack_instance = list(edb.core_padstack.padstack_instances.values())[i]
+            result = edb.core_primitives.convert_padstack_pad_to_rectangle(padstack_instance, "s")
```

Comment:
I would not call the method "convert_pad_to_rectangle" but maybe "create_reactangle_in_pad" and move it in padstactinstances class

Reply:
Good idea. Method renamed and moved.